src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -120,32 +124,62 @@ const Content = ({
                     <ProjectBadge project={project} avatarSize={16} />
                   </Column>
 
-                  <AdoptionColumn>
-                    {showPlaceholders ? (
-                      <StyledPlaceholder width="150px" />
-                    ) : defined(adoption) ? (
-                      <AdoptionWrapper>
-                        <ProgressBarWrapper>
-                          <Tooltip
-                            containerDisplayMode="block"
-                            title={
-                              <AdoptionTooltip
-                                totalUsers={totalUsers}
-                                totalSessions={totalSessions}
-                                totalUsers24h={totalUsers24h}
-                                totalSessions24h={totalSessions24h}
-                              />
-                            }
-                          >
-                            <ProgressBar value={Math.ceil(adoption)} />
-                          </Tooltip>
-                        </ProgressBarWrapper>
-                        <Count value={totalUsers24h ?? 0} />
-                      </AdoptionWrapper>
-                    ) : (
-                      <NotAvailable />
-                    )}
-                  </AdoptionColumn>
+                  {activeDisplay === DisplayOption.CRASH_FREE_SESSIONS &&
```

Comment:
should we update this logic to use these components only once?

Reply:
```suggestion
                 const progressBarValue = supportsSessionAdoption
              ? sessions_adoption
              : adoption;
              
              {activeDisplay === DisplayOption.CRASH_FREE_SESSIONS && (
                    <AdoptionColumn>
                      {showPlaceholders ? (
                        <StyledPlaceholder width="150px" />
                      ) : defined(progressBarValue) ? (
                        <AdoptionWrapper>
                          <ProgressBarWrapper>
                            <Tooltip
                              containerDisplayMode="block"
                              title={
                                <AdoptionTooltip
                                  totalUsers={totalUsers}
                                  totalSessions={totalSessions}
                                  totalUsers24h={totalUsers24h}
                                  totalSessions24h={totalSessions24h}
                                />
                              }
                            >
                              <ProgressBar value={Math.ceil(progressBarValue)} />
                            </Tooltip>
                          </ProgressBarWrapper>
                          <Count value={totalUsers24h ?? 0} />
                        </AdoptionWrapper>
                      ) : (
                        <NotAvailable />
                      )}
                    </AdoptionColumn>
                  )}
```

🤷‍♀️

Reply:
Thanks, needed few adjustments so I committed my own thing.

I would still like to make a clean-up around these files in the near future.